.circleci/config.yml

```diff
@@ -47,24 +47,17 @@ jobs:
               exit 1
             fi
 
-      - run:
-          name: go vet
-          working_directory: test/acceptance
-          command: go vet ./...
-
       - run:
           name: lint-consul-retry
           working_directory: test/acceptance
           command: |
-            go get -u github.com/hashicorp/lint-consul-retry
+            go install github.com/hashicorp/lint-consul-retry@latest
             lint-consul-retry
 
       - run:
           name: golangci-lint
           working_directory: test/acceptance
-          command: |
-            curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.31.0
```

Comment:
Removed golangci-lint install here, because it is already included in the consul-ecs-test:0.3.3 image